<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QWidget" name="Admin">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1044</width>
    <height>700</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
    <stylestrategy>PreferDefault</stylestrategy>
   </font>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="toolTipDuration">
   <number>-1</number>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>570</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>750</width>
     <height>250</height>
    </rect>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="getTable">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>570</y>
     <width>40</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="submitChanges">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>640</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Подтвердить</string>
   </property>
   <property name="icon">
    <iconset>
     <selectedon>:/icon/main/checkmark.png</selectedon>
    </iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteSelectedRows">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>640</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Удалить строку</string>
   </property>
   <property name="icon">
    <iconset>
     <selectedon>:/icon/main/trash.png</selectedon>
    </iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="updatePage">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>640</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Обновить</string>
   </property>
   <property name="icon">
    <iconset>
     <selectedon>:/icon/main/renew.png</selectedon>
    </iconset>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>570</y>
     <width>331</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Вывести информацию о всех пользователях</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Вывести письма всех пользователей</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Вывести информацию о пользователе с логином:</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Вывести письма пользователя с логином:</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="unblockAll">
   <property name="geometry">
    <rect>
     <x>819</x>
     <y>430</y>
     <width>171</width>
     <height>30</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Разблокировать всех</string>
   </property>
   <property name="icon">
    <iconset>
     <selectedon>:/icon/main/unlockAll.png</selectedon>
    </iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="verifyAll">
   <property name="geometry">
    <rect>
     <x>819</x>
     <y>330</y>
     <width>171</width>
     <height>30</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Подтвердить всех</string>
   </property>
   <property name="icon">
    <iconset>
     <selectedon>:/icon/main/checkmark.png</selectedon>
    </iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteAllMessages">
   <property name="geometry">
    <rect>
     <x>819</x>
     <y>530</y>
     <width>171</width>
     <height>30</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Удалить все письма</string>
   </property>
   <property name="icon">
    <iconset>
     <selectedon>:/icon/main/trash.png</selectedon>
    </iconset>
   </property>
  </widget>
  <widget class="QLabel" name="unverifiedUsers">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>300</y>
     <width>171</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Всего на подтверждении:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="blockedUsers">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>400</y>
     <width>181</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Всего заблокировано:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="totalMessages">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>500</y>
     <width>181</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Всего писем в системе:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QFrame" name="donutPieChart">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>0</y>
     <width>375</width>
     <height>300</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: #212427;
</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
   </layout>
  </widget>
  <widget class="QFrame" name="verticalBarChart">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>0</y>
     <width>375</width>
     <height>300</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: #212427;</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
   </layout>
  </widget>
  <widget class="QPushButton" name="backUp">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>599</y>
     <width>141</width>
     <height>33</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>PreferDefault</stylestrategy>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="toolTip">
    <string notr="true"/>
   </property>
   <property name="toolTipDuration">
    <number>0</number>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="whatsThis">
    <string notr="true"/>
   </property>
   <property name="accessibleName">
    <string notr="true"/>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>BackUp</string>
   </property>
   <property name="icon">
    <iconset>
     <selectedon>:/icon/main/downArrow.svg</selectedon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>14</width>
     <height>14</height>
    </size>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRepeatDelay">
    <number>301</number>
   </property>
   <property name="autoRepeatInterval">
    <number>99</number>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="importButton">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>600</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
   <property name="icon">
    <iconset>
     <selectedon>:/icon/main/downArrow.svg</selectedon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>14</width>
     <height>14</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
